src/runmode-dpdk.h

```diff
@@ -31,6 +31,7 @@ typedef struct DPDKIfaceConfigAttributes_ {
     const char *checksum_checks;
     const char *checksum_checks_offload;
     const char *mtu;
+    const char *vlan_strip_enabled;
```

Comment:
I would suggest this to be renamed to the same YAML attribute as it is holding value of so 
`vlan-strip-offload`